src/remarx/app/quote_finder.py

```diff
@@ -155,6 +158,70 @@ def _(mo, original_csv_browser, reuse_csv_browser):
     return original_csvs, reuse_csvs
 
 
+@app.cell
+def _(mo, pathlib, pl, original_csvs):
+    def summarize(selection) -> dict | None:
+        """Build lightweight summary for a selected corpus CSV."""
+        path = pathlib.Path(selection.path)
+        if not path.is_file():
+            return None
+
+        # Use lazy scan to keep memory low; only collect simple aggregates.
+        lf = pl.scan_csv(path, infer_schema_length=0)
+
+        # total sentences
+        total = lf.select(pl.len()).collect().item()
+
+        # section breakdowns when available; otherwise count everything as text
+        if "section_type" in lf.collect_schema():
+            counts = (
+                lf.group_by("section_type")
+                .agg(pl.len().alias("sentences"))
+                .collect()
+            )
+            body = int(
+                counts.filter(pl.col("section_type") == "text")["sentences"].sum()
+            )
+            foot = int(
+                counts.filter(pl.col("section_type") == "footnote")["sentences"].sum()
+            )
+        else:
+            body, foot = total, 0
+
+        # last modified timestamp for display
+        last_updated = datetime.fromtimestamp(path.stat().st_mtime).strftime(
+            "%Y-%m-%d %H:%M:%S"
+        )
+
+        return {
+            "corpus": path.name,
+            "total_lines": total,
+            "body_text_lines": body,
+            "footnote_lines": foot,
+            "last_updated": last_updated,
```

Comment:
Since these are meant for human consumption, I suggest leaving the `_` out to make it more readable.